pkg/controllers/management/drivers/nodedriver/machine_driver.go

```diff
@@ -179,7 +181,22 @@ func (m *Lifecycle) download(obj *v3.NodeDriver) (*v3.NodeDriver, error) {
 		return obj, errs.Wrap(err, "failed JSON in addUIHintsAnno")
 	}
 
-	flags, err := getCreateFlagsForDriver(driverName)
+	var flags []cli.Flag
+	// NOTE(cmurphy): There is not currently a real google machine driver, but
+	// we still want to be able to create a Google cloud credential to use with
+	// GKE. This fake driver flag allows us to go through the motions of
+	// handling the credential fields without actually needing to run the
+	// machine driver binary.
+	if driverName == "google" {
+		flags = []cli.Flag{
+			&cli.StringFlag{
+				Name: "google-auth-encoded-json",
+			},
+		}
```

Comment:
Can we move this inside the `getCreateFlagsForDriver` call? We still want the flags for the "driver" so shortcut in there instead of here.

Reply:
done